relayer/src/chain/cosmos.rs

```diff
@@ -2026,13 +2026,29 @@ impl ChainEndpoint for CosmosSdkChain {
                 sequence,
             }
             .into(),
-            PacketMsgType::TimeoutUnordered => ReceiptsPath {
-                port_id,
-                channel_id,
-                sequence,
-            }
-            .into(),
-            PacketMsgType::TimeoutOrdered => SeqRecvsPath(port_id, channel_id).into(),
+            PacketMsgType::TimeoutUnordered => {
+                debug!(
+                    "XXXX getting UNORDERED timout proof with SeqRecvsPath {}, {}, {}",
```

Comment:
Cleaned this up a bit using structured logging in 8ab1a72a11e371114c64f24eca48f6ad8711e6cc.

Reply:
yes, sorry this was not supposed to be there. part of a an in-progress debug session that i was doing on that branch.